app/controllers/admin/api_docs/services_controller.rb

```diff
@@ -101,11 +100,19 @@ def api_docs_params(*extra_params)
     params.require(:api_docs_service).permit(*permit_params)
   end
 
+  def set_service_navigation_if_service
```

Comment:
I think you should be able to use the `:if` options

https://github.com/3scale/porta/blob/master/lib/menu_system.rb#L111